src/SFML/Window/GlResource.cpp

```diff
@@ -28,6 +28,7 @@
 #include <SFML/Window/GlContext.hpp>
 #include <SFML/Window/GlResource.hpp>
 
+#include <utility>
```

Comment:
```suggestion
#include <utility>

```
To preserve the two empty lines between the last header and `namespace sf`